test_data/nixpkgs_repository/lib_customisation.good.nix

```diff
@@ -17,7 +17,5 @@
           crossDrv = overrideDerivation drv.crossDrv f;
           nativeDrv = overrideDerivation drv.nativeDrv f;
         }
-        else { }
-      )
-    );
+        else { }));
```

Comment:
This change somehow related to https://github.com/nix-community/nixpkgs-fmt/issues/184. if we want to make `NODE_IF` inline with `(`, then it will caused uneven indentation:

```
(if (drv ? crossDrv && drv ? nativeDrv)
  then {
    crossDrv = overrideDerivation drv.crossDrv f;
    nativeDrv = overrideDerivation drv.nativeDrv f;
  }
  else { }));
```

Reply:
>then it will caused uneven indentation:

we might say that we want `if` to be anchor for `then` in this case somehow?

Reply:
That is what I am thinking because we want to make `if`, `then`, `else` align each other, no? 

So if our indentation value is 2 spaces and then we make `if` inline with `(`, both `then` and `else` have 1 additional space.